app/src/main/java/com/willowtree/vocable/presets/PresetCategories.kt

```diff
@@ -0,0 +1,38 @@
+package com.willowtree.vocable.presets
+
+import com.willowtree.vocable.R
+
+enum class PresetCategories(val id: String) {
+    GENERAL("preset_general_category_id"),
+    BASIC_NEEDS("preset_basic_needs_category_id"),
+    CONVERSATION("preset_conversation_category_id"),
+    ENVIRONMENT("preset_environment_category_id"),
+    PERSONAL_CARE("preset_personal_care_category_id"),
+    USER_KEYPAD("preset_user_keypad"),
+    USER_FAVORITES("preset_user_favorites");
+
+
+    fun getArrayId(): Int {
+        return when (this) {
+            GENERAL -> R.array.category_general
+            BASIC_NEEDS -> R.array.category_basic_needs
+            CONVERSATION -> R.array.category_conversation
+            ENVIRONMENT -> R.array.category_environment
+            PERSONAL_CARE -> R.array.category_personal_care
+            USER_KEYPAD -> -1 // Not localized with same convention
```

Comment:
Is the keypad static or could it vary by language? Could potentially move these to use the same format as the rest of the presets

Reply:
I'm pretty sure it's static, the only thing that should change in the keypad is "yes" and "no"

Reply:
Ok perfect, I can move this to support the new pattern then. Thanks!